client/src/app/core/repositories/assignments/assignment-candidate-repository.service.ts

```diff
@@ -14,7 +13,7 @@ import { RepositoryServiceCollector } from '../repository-service-collector';
 @Injectable({
     providedIn: 'root'
 })
-export class AssignmentCandidateRepositoryService extends MeetingModelBaseRepository<
+export class AssignmentCandidateRepositoryService extends BaseRepositoryWithActiveMeeting<
```

Comment:
Is this a forecast into the future (You know that you will need it), or why did you change it?